interop-testing/src/main/java/io/grpc/testing/integration/TestServiceServer.java

```diff
@@ -130,11 +138,14 @@ void parseArgs(String[] args) {
               + "\n  --local_handshaker_port=PORT"
               + "\n                        Use local ALTS handshaker service on the specified port "
               + "\n                        for testing. Only effective when --use_alts=true."
+              + "\n  --address_type=IPV4|IPV6|IPV4_IPV6"
```

Comment:
Will there be an update to https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md ?